src/components/JobConfiguration.vue

```diff
@@ -328,7 +416,17 @@ ion-list {
 
   .mobile-only {
     display: none;
-  }  
+  }
+  .more-actions {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: end;
+    row-gap: var(--spacer-sm);
+    margin-top: var(--spacer-sm);
+  }
+  .more-actions > ion-item {
```

Comment:
we shouldn't specify ion item, it should just be all items in more-actions

Reply:
Made the requested change.